src/ResourceList.scss

```diff
@@ -144,86 +159,89 @@ $borderRadius: 5px;
     flex: auto;
   }
 
-  &:not(:last-child) {
-    border-bottom: 1px solid rgba(37,59,115,0.1);
+  &:last-child {
+    margin-bottom: 1.2rem;
```

Comment:
margin-inline/block is used elsewhere. Should it be used here as well?

Reply:
Could be used here, but then I think we should do it in the rest of the file as well.

Reply:
Definitely. The mix currently looks intentional while it's not. Would be nice to clean up. In another PR maybe.